bark/python_wrapper/world/map.cpp

```diff
@@ -26,6 +26,9 @@ using bark::world::opendrive::XodrLaneId;
 void python_map(py::module m) {
   py::class_<MapInterface, std::shared_ptr<MapInterface>>(m, "MapInterface")
       .def(py::init<>())
+      .def_property("add_full_junction_area",
```

Comment:
I'm a little bit confused about the naming. is it really a property? then remove "add"?

Reply:
its a boolean flag if the junction area shall be added to the road corridor. I renamed it.